src/main/java/org/fogbowcloud/saps/engine/core/catalog/jdbc/JDBCCatalogUtil.java

```diff
@@ -0,0 +1,47 @@
+package org.fogbowcloud.saps.engine.core.catalog.jdbc;
+
+import org.fogbowcloud.saps.engine.core.catalog.exceptions.CatalogException;
+import org.fogbowcloud.saps.engine.core.model.SapsImage;
+import org.fogbowcloud.saps.engine.core.model.SapsUser;
+import org.fogbowcloud.saps.engine.core.model.enums.ImageTaskState;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JDBCCatalogUtil {
+
+    public static SapsUser extractSapsUser(ResultSet rs) throws CatalogException {
+        SapsUser sebalUser = null;
+        try {
+            sebalUser = new SapsUser(rs.getString(JDBCCatalogConstants.Tables.User.EMAIL), rs.getString(JDBCCatalogConstants.Tables.User.NAME),
+                    rs.getString(JDBCCatalogConstants.Tables.User.PASSWORD), rs.getBoolean(JDBCCatalogConstants.Tables.User.ENABLE), rs.getBoolean(JDBCCatalogConstants.Tables.User.NOTIFY),
+                    rs.getBoolean(JDBCCatalogConstants.Tables.User.ADMIN_ROLE));
+        } catch (SQLException e) {
+            throw new CatalogException("Error while extract user");
```

Comment:
Add the cause exception on exception's constructor.

Reply:
Done